frontend/src/components/Burger/index.jsx

```diff
@@ -13,16 +14,29 @@ function Burger() {
   return (
     <SBurger toggleNav={toggleNav} open={open}>
       <img src={logoAdangris} alt="logo" className="logo" />
+
       {toggleMenu && (
         <ul>
-          <li className="picture">Page d'accueil</li>
-          <li className="picture">Mon profil</li>
-          <li className="picture">Mon agenda</li>
-          <li className="picture">A propos</li>
-          <li className="picture">Déconnexion</li>
+          <Link to="/profile/:profileId">
+            <li className="picture">Mon profil</li>
+          </Link>
+          <Link to="/">
+            <li className="picture">Page d'accueil</li>
+          </Link>
+          <Link to="/agenda">
+            <li className="picture">Mon agenda</li>
+          </Link>
+          <Link to="/about">
+            <li className="picture">A propos</li>
+          </Link>
+          <Link to="/logout">
+            <li className="picture">Déconnexion</li>
+          </Link>
```

Comment:
Not a blocking one, but direct children of `<ul>` should be `<li>`, not `<Link>`